### DISCUSSION TITLE
Risk Coverage filters - enhancement to show failing CodeQL

### DISCUSSION THREAD

**Discussion by Josephpul** [community/community#63711] [Code Security] [answered]
### Select Topic Area

Product Feedback

### Body

Hi,

The [Security Coverage](https://github.com/orgs/Underwriters-Labs/security/coverage) page is great for viewing which repositories have things like 'Code Scanning' enabled. However, it would be great if it also could show which repositories have CodeQL actions failing. I realize it will\can run on multiple branches so perhaps just against the default branch? Perhaps adding this in as a new filter or indicator on the page if there is failure would then give this flexibility.

**Comment by AlonaHlobina** [comment, marked as answer]
Hello @Josephpul,
Thank you for sharing your feedback with us. We appreciate your suggestion and would like to inform you that our team is currently considering adding this functionality to the security coverage page. We understand the importance of this feature and will work towards delivering it in the near future. Your interest and support are highly valued. 🌞
